com.github.arminstraub.krop.yaml

```diff
@@ -0,0 +1,177 @@
+---
+app-id: com.github.arminstraub.krop
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: krop
+
+finish-args:
+  - --env=PATH=/app/bin:/usr/bin
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  # Without dri Qt prints "libEGL warning: wayland-egl: could not open /dev/dri/card0"
+  - --device=dri
+
+modules:
+  - org.boost.Boost.json
+  - qtbase-5-12.yml
```

Comment:
why do you need to build qt5?

Reply:
This is a Qt app (through PyQt bindings). Does Freedesktop 21.08 runtime include Qt?

Reply:
that's what the KDE runtime is for.

Reply:
Well this app is not using any kde api, so the flatpak will be more light without the KDE runtime, especially if you're a GNOME user.

Same way there are several GTK apps based on Freedesktop runtime because the don't need the GNOME one.

Reply:
if you need Qt, use the KDE runtime.